tools/detd_docker_build.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+IMAGE_NAME="detd_builder"
+CONTAINER_NAME="detd_build_container"
+DEB_DIRECTORY="/tmp/detd"
+
+
+function build_detd_deb {
+
+    # Temperory folder to copy detd to conatiner
+    # Due to security access reasons, ADD command not able to access any other folder.
+    # Clean up old directory
+    rm -rf tmp_detd
```

Comment:
Please replace tmp_detd here and in the remainder of the script by a variable, e.g. TMP_DIRECTORY, for consistency. This may also be helpful to keep different temporary directories in place if build containers for other distributions are added and we trigger multiple builds at once for QA.

You may also move the explanation about ADD above with the variable definition.